build-tools/xa-prep-tasks/Xamarin.Android.BuildTools.PrepTasks/JdkInfo.cs

```diff
@@ -67,6 +77,10 @@ public override bool Execute ()
       </ItemGroup>
     </When>
   </Choose>
+  <PropertyGroup>
```

Comment:
...with the above `<ReplaceFileContents/>` approach in mind, you could *remove* this `File.WriteAlltext()` block here, and replace with an approach of additional `[Output]` properties within `JdkInfo` + a `<ReplaceFileContents/>` call.